be/src/exec/except_node.cpp

```diff
@@ -179,17 +137,21 @@ Status ExceptNode::get_next(RuntimeState* state, RowBatch* out_batch, bool* eos)
     if (reached_limit()) {
         return Status::OK();
     }
+    int64_t tuple_buf_size;
+    uint8_t* tuple_buf;
+    RETURN_IF_ERROR(
+            out_batch->resize_and_allocate_tuple_buffer(state, &tuple_buf_size, &tuple_buf));
     uint32_t previous_hash = -1;
     TupleRow* previous_row = nullptr;
     while (_hash_tbl_iterator.has_next()) {
+        VLOG_ROW << "find row: "
+                 << get_row_output_string(_hash_tbl_iterator.get_row(), child(0)->row_desc())
+                 << " matched: " << _hash_tbl_iterator.matched();
         if (!_hash_tbl_iterator.matched()) {
             if (previous_hash != _hash_tbl_iterator.get_hash() ||
                 !equals(previous_row, _hash_tbl_iterator.get_row())) {
```

Comment:
why does hashtable has the same value？When you create a hash table, will duplicate values are inserted into the hash table？

Reply:
As I know, this hash table will store all rows with same hash key.

Reply:
But this is a set operation（It is different from join）. The set operation has deduplicated, so the elements of the hash table do not need to have duplicatited value。

Reply:
duplicated values will be iterate when get next,and deduplicate can reduce the cost of probe,you can find detail in hash_table.h,hash table in doris is not same with the common sense hashtable